atd-toolbox/s3_restore_cr3_pdfs.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/python3
+
+from __future__ import print_function
+
+import re
+import os
+import sys
+import json
+import boto3
+import magic
```

Comment:
Brilliant! I've never worked with magic, it's cool to see it in action. 🧙 ✨

Reply:
@frankhereford FYI, the [PEP 8 guidelines](https://www.python.org/dev/peps/pep-0008/#imports) on module imports. To make the dependencies super clear, the best practice is to import 3rd party libs after built-ins. I even add a linebreak between built-in/3rd-party/local imports but that is probably controversial.

Reply:
That's really helpful to have the guidelines document to reference. I also like the new line to group them into logical sections. Thanks for that advice, I'm going to adopt it in the future.